src/reachy2_sdk/subparts/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""
+# Subparts module.
```

Comment:
Not a big fan of the subparts name. If we keep this we should move it inside parts folder. But parts mean more a body part for us.
Why not sensors? We may have more in the future btw